crates/samples/direct3d12/src/main.rs

```diff
@@ -73,18 +73,20 @@ where
         title.push_str(" (WARP)");
     }
 
+    title.push('\0');
+
     let hwnd = unsafe {
         CreateWindowExA(
-            Default::default(),
-            "RustWindowClass",
-            title,
+            WINDOW_EX_STYLE::default(),
+            PCSTR(b"RustWindowClass".as_ptr()),
```

Comment:
Missing a null terminator. `HSTRING` would work if we used `CreateWindowExW` instead...